### PR TITLE
Missing multisignature account details view

### LINKED ISSUE #4621: Missing multisignature account details view
### Expected behavior

Multisignature account design should be implemented:
- [Icon](https://www.figma.com/file/KcrDpvWEKQhdGwNd4CZ5NY/Desktop-Prototype?node-id=38%3A13860&t=CpuFKV5SeBIImhbI-4)
- [View](https://www.figma.com/file/KcrDpvWEKQhdGwNd4CZ5NY/Desktop-Prototype?node-id=38%3A13923&t=CpuFKV5SeBIImhbI-4)

- Ensure the `Register multisignature account` menu item in account is updated to `View multisignature account`
### Actual behavior

Unable to view Multisignature account details

### Steps to reproduce

- Register multisignature account

### Which version(s) does this affect? (Environment, OS, etc...)
v3

### PR DESCRIPTION
**Pull request by reyraa**
### What was the problem?

This PR resolves #4621

### How was it solved?

- Added missing icon in the explorer page
- Added missing UI elements in the dialog view

### How was it tested?

- Visually compared to the designs
- Updated unit tests

### CODE REVIEW

**Review by ManuGowda — CHANGES_REQUESTED** [review]
Screenshot 2023-01-05 at 6 50 02 PM
Screenshot 2023-01-05 at 6 50 57 PM

File: src/modules/wallet/components/multisigAccountDetails/index.js [hunk 1]
```diff
@@ -3,15 +3,19 @@ import { useSelector } from 'react-redux';
...
 import Box from 'src/theme/box';
-import BoxHeader from 'src/theme/box/header';
+// import BoxHeader from 'src/theme/box/header';
+import { useCurrentAccount } from '@account/hooks';
 import BoxContent from 'src/theme/box/content';
 import BoxInfoText from 'src/theme/box/infoText';
 import Dialog from 'src/theme/dialog/dialog';
+import Icon from 'src/theme/Icon';
 import Tooltip from 'src/theme/Tooltip';
+import defaultBackgroundImage from '../../../../../setup/react/assets/images/default-chain-background.png';
```
Comment (oskarleonard) [inline src/modules/wallet/components/multisigAccountDetails/index.js]: Maybe use our aliases? 
`@setup/...`

File: src/modules/wallet/components/multisigAccountDetails/index.js [hunk 2]
```diff
@@ -3,15 +3,19 @@ import { useSelector } from 'react-redux';
...
-import { selectActiveTokenAccount, selectNetwork } from 'src/redux/selectors';
-import routes from 'src/routes/routes';
-import { extractAddressFromPublicKey } from '@wallet/utils/account';
+import { selectNetwork } from 'src/redux/selectors';
+import { extractAddressFromPublicKey, truncateAddress } from '@wallet/utils/account';
+import WalletVisual from '@wallet/components/walletVisual';
+import CopyToClipboard from 'src/modules/common/components/copyToClipboard';
 import Box from 'src/theme/box';
-import BoxHeader from 'src/theme/box/header';
+// import BoxHeader from 'src/theme/box/header';
```
Comment (oskarleonard) [inline src/modules/wallet/components/multisigAccountDetails/index.js]: comment

File: src/modules/wallet/components/walletVisualWithAddress/index.js [hunk 3]
```diff
@@ -7,14 +7,59 @@ import PropTypes from 'prop-types';
...
+const AccountAddress = ({
+  address,
+  truncate,
+  copy,
+  transformedAddress,
+  truncatedAddress,
+  name,
+  isMultisig,
+}) => (
+  <span className={`${styles.address} accountAddress`}>
```
Comment (oskarleonard) [inline src/modules/wallet/components/walletVisualWithAddress/index.js]: We cannot have divs inside spans, both CopyToClipboard and DialogLink are divs

**Review by oskarleonard — DISMISSED** [review]

**Review by oskarleonard — APPROVED** [review]

**Review by ManuGowda — APPROVED** [review]

**Pull request merged**